src/commtrack.js

```diff
@@ -136,7 +137,9 @@ define([
             isHashtaggable: false,
             spec: {
                 date: {
-                    visibility: 'hidden',
+                    lstring: 'Date',
+                    help: 'The date of the action',
```

Comment:
This is just a textbox, isn't it? If it needs a particular format, should mention that in the help.